gms/gossiper.cc

```diff
@@ -947,44 +947,32 @@ future<> gossiper::replicate_live_endpoints_on_change() {
     //
     // Gossiper task runs only on CPU0:
     //
-    //    - If endpoint_state_map or _live_endpoints have changed - replicate
```

Comment:
The "shadow" copies were used to safely replicate these variables to other shards, while allowing the original variables to be modified on shard 0 in parallel (avoiding a data race).

Do I understand correctly that right now the parallel modification possibility is gone, because we always modify and replicate under the lock?

Reply:
correct